var/spack/repos/builtin/packages/py-flake8/package.py

```diff
@@ -57,11 +57,12 @@ class PyFlake8(Package):
     # mccabe >= 0.2.1, < 0.5
     depends_on('py-mccabe@0.2.1:0.4.0', when='@2.5.4', type=('build', 'run'))
 
-    depends_on('py-configparser', when='^python@:3.3.999', type=('build', 'run'))
-    depends_on('py-enum34', when='^python@:3.1.999', type=('build', 'run'))
+    # These dependencies breaks concretization
+    # See https://github.com/LLNL/spack/issues/2793
+    # depends_on('py-configparser', when='^python@:3.3.999', type=('build', 'run'))  # noqa
+    # depends_on('py-enum34', when='^python@:3.1.999', type=('build', 'run'))
```

Comment:
I don't believe that's the case. You need to use a range. `3.3` only matches `3.3`, so if you tried to install something that depends on `python@3.3`, it will try to fetch `python-3.3.tar.gz`, which won't exist.